BAC0/scripts/Base.py

```diff
@@ -83,6 +83,8 @@ def __init__(
         self._started = False
         self._stopped = False
 
+        self._log.info(f'IP: {localIPAddr} / Type: {type(localIPAddr)}')
```

Comment:
I don't know why, but I have been getting a lot of the following:

```
beaconworker: ERROR    update_device_points: IP Address provided (192.168.1.254) invalid.
Traceback (most recent call last):
  File "/etc/beacon/beacon_web/batch/task.py", line 33, in wrapper
    _connection = bactalk.session()
  File "/etc/beacon/lib/bactalk.py", line 24, in session
    return BAC0.lite(**params)
  File "/etc/beacon/venv/lib/python3.6/site-packages/BAC0/scripts/Lite.py", line 102, in __init__
    **params
  File "/etc/beacon/venv/lib/python3.6/site-packages/BAC0/scripts/Base.py", line 95, in __init__
    "IP Address provided ({}) invalid.".format(localIPAddr)
BAC0.core.io.IOExceptions.InitializationError: IP Address provided (192.168.1.254) invalid.
```

Adding this log seems to fix the code execution

:thinking:

Reply:
We'll need more details...
Please don't use f'string for now. It would break compatibility with version before 3.7

Reply:
I pushed a change to remove any f-strings

Reply:
Also, I think the root cause of this was mistakenly attempting to make a duplicate BAC0 connection when there was one already present.